assets/js/state/selectors/cluster.js

```diff
@@ -10,6 +10,13 @@ export const getClusterHostIDs =
       .filter((host) => host.cluster_id === clusterID)
       .map(({ id: hostID }) => hostID);
 
+export const getClusterHostNames =
```

Comment:
The name is a bit misleading. Maybe we could have `getClusterHosts` and return all the fields?

Reply:
I followed the train of thoughts of the previous `getClusterHostIDs`, which might need to be revisited as well based on this suggestion.

If it's not a hard requirement changing now, I'd keep it like this and iterate over at a later stage :sweat_smile: